config/prod/wpoehlm-largevol-ec2-ebs-vol.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision an EBS volume for wpoehlm-ios-ec2 instance
+template_path: remote/managed-ebs.yaml
+stack_name: wpoehlm-iops-ec2-ebs-vol
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "NDR"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: "william.poehlman@sagebase.org"
+
+  AvailabilityZone: "us-east-1a"
+  VolumeType: "io1"
+  Size: "1280"
+  Iops: "64000"
+  Ec2InstanceId: "i-0960d784a9185f87e"
```

Comment:
this instance id is for `wpoehlm-parallel-ec2` instance.  it will need to be updated since the instance was removed by PR https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/131

Reply:
When I look at the wpoehlm-iops-ec2 instance (the one we want to attach this volume to), the instance ID is the same as here (i-0960d784a9185f87e)

Reply:
oops.  you are correct.  I got confused.  no need to update